### PR TITLE
Add a "Now" button for timestamp

### PR DESCRIPTION
**Pull request by kohenkatz**
I often want to set a timestamp field to the current time, and have to do that manually.

This PR adds a button next to the `google.protobuf.Timestamp` fields to set the timestamp to the current UTC time (based on the browser's time).

### CODE REVIEW

**Comment by kohenkatz** [comment]
@dariusdamalakas It does not look like any members of the project team have looked at this pull request. It does look like a new build of the `bindata.go` file will be needed in order to merge this now - I will see if I can do that today, and hopefully someone who can merge it will have a look (and either merge it or explain why they choose not to merge it).

**Comment by kohenkatz** [comment]
This PR is once again ready to merge, now that conflicts have been resolved.

**Comment by dariusdamalakas** [comment]
FYI @dragonsinth

**Review by dragonsinth — APPROVED** [review]

**Pull request merged**
